pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -10,6 +9,7 @@
 
 from .git_provider import FilePatchInfo
 
+
 class BitbucketProvider:
     def __init__(self, pr_url: Optional[str] = None):
         s = requests.Session()
```

Comment:
Consider using a more descriptive variable name than 's'. This would make the code more readable and maintainable. [medium]